Controller subscriptions

saulomendonca opened the pull request:
issue #12 
Added the controller to manage subscriptions
Added the permission control for this subscriptions controller

lairtonmendes commented:
I believe this can stay in the ability.rb
```
def self.can_approve?(institution, user)
    profiles_with_permissions = ["owner", "admin", "approver"]
    profiles_user = where(institution: institution, user: user).map(&:profile) 
    (profiles_with_permissions & profiles_user).any?
end

```
I think we can use this pattern in ability.rb
[CanCanCan](https://github.com/CanCanCommunity/cancancan/wiki/Role-Based-Authorization#alternative-role-inheritance)

saulomendonca commented:
@lairtonmendes, I agree with the logic in ability.rb. But I was not able to apply the "Role Inheritance Pattern" that you mentioned is this context. In our application, there is no 'generic' role, but the permission depends on the association institution. 
So, I moved the logic to the ability.rb and added tests.
